Library/Formula/harfbuzz.rb

```diff
@@ -34,6 +34,10 @@ class Harfbuzz < Formula
     sha256 "9535d35dab9e002963eef56757c46881f6b3d3b27db24eefcc80929781856c77"
   end
 
+  def patches
+    DATA
```

Comment:
Is it OK if the comment points to the issue upstream that has a patch in a comment?
```ruby
  # See https://github.com/behdad/harfbuzz/issues/183
  patch do
    DATA
  end
```